packages/preload/vite.config.js

```diff
@@ -2,6 +2,8 @@ import {chrome} from '../../.electron-vendors.cache.json';
 import {preload} from 'unplugin-auto-expose';
 import {join} from 'node:path';
 import {injectAppVersion} from '../../version/inject-app-version-plugin.mjs';
+import nodeBuiltins from 'builtin-modules/static';
```

Comment:
Why are you use this instead of [`node:modules`](https://nodejs.org/docs/latest-v16.x/api/module.html#the-module-object)

Reply:
Ah that's better. Will fix later.

Reply:
I've removed both builtin-modules and electron-builtins. The latter just wraps submodule under `electron` so I just need to externalize `electron`.